docs/ci/ci-automated-release-workflow.md

```diff
@@ -0,0 +1,134 @@
+# Creating Automated Releases: Design Document
+
+## Motivation & Overview
+
+Presently, the release processes for every library within this `instructlab` GitHub organization is entirely manual:
+
+For example, for a typical y-stream release, a maintainer has to:
+
+1. Manually create a new release branch -- e.g., `release-0.y.0`,
+2. Manually create a pull request against `release-0.y.0` to cap the versions on some of the dependencies defined in their library's `requirements.txt` file,
```

Comment:
```suggestion
2. Manually create a pull request against the `release-0.y` branch to cap the versions on some of the dependencies defined in their library's `requirements.txt` file
```

Reply:
Thanks for catching that again will update.